lib/cli/src/commands/whoami.rs

```diff
@@ -12,9 +12,24 @@ impl Whoami {
     /// Execute `wasmer whoami`
     pub fn execute(&self) -> Result<(), anyhow::Error> {
         let registry = self.env.registry_endpoint()?;
-        let (registry, username) =
-            wasmer_registry::whoami(self.env.dir(), Some(registry.as_str()), None)?;
-        println!("logged into registry {registry:?} as user {username:?}");
+
+        match wasmer_registry::current_user(self.env.dir(), Some(registry.as_str()), None)? {
+            Some(CurrentUser {
+                registry,
+                user,
+                verified,
+                ..
+            }) => {
+                println!("logged into registry \"{registry}\" as user \"{user}\"");
+                if !verified {
+                    println!("Warning: Your email address still needs to be verified");
```

Comment:
```suggestion
                    eprintln!("Warning: Please verify your email address to fully access all Wasmer features");
```

I am not familiar with the codebase so maybe using eprintln isn't the right way? idk, just a suggestion.